scripts/import_cloudwatch_log_groups.py

```diff
@@ -0,0 +1,83 @@
+import json
+
+import boto3
+from more_itertools import (
+    one,
+)
+
+from azul import (
+    config,
+    logging,
+)
+from azul.logging import (
+    configure_script_logging,
+)
+from azul.terraform import (
+    terraform,
+)
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    def add_prefix(name):
```

Comment:
It's not at all clear what this function does based on its name. Personally I prefer `qualify_log_group_name`, but it's a bit verbose. Maybe just `log_group_name` would be okay? Something to think about.